web/src/pages/salesforce/index.js

```diff
@@ -0,0 +1,263 @@
+import React from "react";
+import Layout from "../../components/Layout/Layout";
+import { Container, Col, Row, Image } from "react-bootstrap";
+import { GatsbyImage, StaticImage } from "gatsby-plugin-image";
+import Title from "../../components/UI/Title/Title";
+import Subtitle from "../../components/UI/Subtitle/Subtitle";
+import BrandButton from "../../components/UI/BrandButton/BrandButton";
+import "../../styles/main.scss"
+import styled from "styled-components";
+import Accordion from 'react-bootstrap/Accordion';
+import "./salesforce.scss";
+
+const SalesforcePage = ({ data }) => {
+    
+    return (
+        <Layout pageTitle="Salesforce | Awesome Inc">
+            
+            {/* Header section */}
+            <section id="header">
+                <Container fluid>
+                    <Row className="header-section">
+                        <Col className="header-text">
+                            <Title className="text-uppercase">Salesforce career accelerator</Title>
+                            <BrandButton className="my-3">START YOUR APPLICATION</BrandButton>
```

Comment:
@Sisay-su Looks like there are still places that have all-caps. Can you remove all of them on this page by using the `text-uppercase` class?